package.json

```diff
@@ -30,6 +30,7 @@
     "pg": "^8.9.0",
     "pg-connection-string": "^2.5.0",
     "postgres-migrations": "^5.3.0",
+    "prettier": "^2.8.8",
```

Comment:
OOC: Why move this from dev to prod dependencies?

Reply:
I'm not quite sure why it's in there. I can move it back to dev dependencies.